provisioning/provision.sh

```diff
@@ -59,16 +59,16 @@ systemctl restart elasticsearch
 # We install some dev tools (tmux and vim)
 apt-get install -y tmux vim-nox
 # We create a virtualenv for Dissemin
-python3 -m venv /dissemin/.vm_venv
+python3 -m venv /home/vagrant/.vm_venv
```

Comment:
I got weird issues with `pip` when running `--upgrade` in the VM, `EnvironmentErrors` and issues with deleting cache folders from pip (starting with a `~`). I guess this is because the virtualenv and the special characters was living in a shared folder, it seems to be fine with the virtualenv on the VM disk only.